app/experimenter/experiments/tests/api/v5/test_query.py

```diff
@@ -107,3 +111,83 @@ def test_experiment_by_slug_not_found(self):
         content = json.loads(response.content)
         experiment = content["data"]["experimentBySlug"]
         self.assertIsNone(experiment)
+
+    def test_nimbus_config(self):
+        user_email = "user@example.com"
+        # Create some probes and feature configs
+        feature_configs = NimbusFeatureConfigFactory.create_batch(10)
+        for feature_config in feature_configs:
+            feature_config.save()
+        probe_sets = NimbusProbeSetFactory.create_batch(10)
+        for probe_set in probe_sets:
+            probe_set.save()
+
```

Comment:
It looks like `create_batch` persists to the database, so I don't think you need to call `save` here at all?

```py
In [6]: NimbusProbeSetFactory.create_batch(10)
Out[6]: 
[<NimbusProbeSet: Inverse encompassing superstructure>,
 <NimbusProbeSet: Vision-oriented next generation support>,
 <NimbusProbeSet: Assimilated asynchronous emulation>,
 <NimbusProbeSet: Seamless disintermediate standardization>,
 <NimbusProbeSet: Cloned responsive framework>,
 <NimbusProbeSet: Profit-focused modular capacity>,
 <NimbusProbeSet: Digitized web-enabled frame>,
 <NimbusProbeSet: Profit-focused radical superstructure>,
 <NimbusProbeSet: Decentralized bifurcated customer loyalty>,
 <NimbusProbeSet: Secured heuristic architecture>]

In [7]: NimbusProbeSet.objects.all()
Out[7]: <QuerySet [<NimbusProbeSet: Inverse encompassing superstructure>, <NimbusProbeSet: Vision-oriented next generation support>, <NimbusProbeSet: Assimilated asynchronous emulation>, <NimbusProbeSet: Seamless disintermediate standardization>, <NimbusProbeSet: Cloned responsive framework>, <NimbusProbeSet: Profit-focused modular capacity>, <NimbusProbeSet: Digitized web-enabled frame>, <NimbusProbeSet: Profit-focused radical superstructure>, <NimbusProbeSet: Decentralized bifurcated customer loyalty>, <NimbusProbeSet: Secured heuristic architecture>]>
```

Reply:
Did you force push the wrong thing?  I still see the save lines?

Reply:
Odd, pushed again, that should do it.